Integrations/integration-IntSights.yml

```diff
@@ -1584,7 +1663,15 @@ script:
       description: The alert's closed reason
       type: string
     description: Close an alert
+  - name: intsights-mssp-get-sub-accounts
+    arguments: []
+    description: Get all mssp sub accounts
+  - name: intsights-mssp-test
+    arguments: []
+    description: Tests intsights mssp features, should be an mssp account.
```

Comment:
Need to add outputs - `IntSights.MsspAccount.ID`, `IntSights.MsspAccount.Status`, etc.